javascript/extractor/src/com/semmle/jcorn/CustomParser.java

```diff
@@ -320,4 +377,240 @@ protected Expression parseNew() {
 		}
 		return res;
 	}
+
+	/*
+	 * E4X
+	 *
+	 * PrimaryExpression :
+	 *     PropertyIdentifier
+	 *     XMLInitialiser
+	 *     XMLListInitialiser
+	 *
+	 * PropertyIdentifier :
+	 *     AttributeIdentifier
+	 *     QualifiedIdentifier
+	 *     WildcardIdent
+	 *
+	 * AttributeIdentifier :
+	 *     @ PropertySelector
+	 *     @ QualifiedIdentifier
+	 *     @ [ Expression ]
+	 *
+	 * PropertySelector :
+	 *     Identifier
+	 *     WildcardIdentifier
+	 *
+	 * QualifiedIdentifier :
+	 *     PropertySelector :: PropertySelector
```

Comment:
This is what's causing the spurious alerts on Lucifier, since it causes an ambiguity with the proposed `::` function-bind operator.